bin/lint

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+IMAGE="arnold:$(tr -d '\n' < VERSION)"
+
+print_green() {
+  echo -e "\e[32m${1}\e[0m"
+}
+
+# Fontions inside Arnold image
+
+_inside_bash() {
+  # https://github.com/koalaman/shellcheck
+  echo -e "\n---> Bash: lint\n"
+  shellcheck --shell=bash bin/*
+  print_green "  -> Bash lint OK"
+}
+
+_inside_ansible(){
+  echo -e "\n---> Ansible: check syntax\n"
+  ansible-playbook --syntax-check ./*.yml
+  print_green "  -> Ansible syntax OK"
+
+  # https://github.com/willthames/ansible-lint
+  echo -e "\n---> Ansible: lint\n"
+  # Skip [E602] Line too long
+  SKIP_LIST="E602"
+  ansible-lint -R -r "${ANSIBLE_LINT_RULES_DIR}" -x $SKIP_LIST ./*.yml
+  print_green "  -> Ansible lint OK"
+}
```

Comment:
I think those `_inside_foo()` functions are a bit confusing. IMO your code should stand in 3 functions: `_docker`, `_ansible` and `_bash` (and `_all`). We expect that these checks will always run in a container _via_ a `docker run` call and it's perfectly fine.